tests/test-cast-to-overflow.c

```diff
@@ -0,0 +1,91 @@
+
+/* Unit test trucate in type cast.
+   Copyright (C) 2016 Free Software Foundation, Inc.
+   This file is part of the Decimal Floating Point C Library.
+   The Decimal Floating Point C Library is free software; you can
+   redistribute it and/or modify it under the terms of the GNU Lesser
+   General Public License version 2.1.
+   The Decimal Floating Point C Library is distributed in the hope that
+   it will be useful, but WITHOUT ANY WARRANTY; without even the implied
+   warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See
+   the GNU Lesser General Public License version 2.1 for more details.
+   You should have received a copy of the GNU Lesser General Public
+   License version 2.1 along with the Decimal Floating Point C Library;
+   if not, write to the Free Software Foundation, Inc., 59 Temple Place,
+   Suite 330, Boston, MA 02111-1307 USA.
+   Please see libdfp/COPYING.txt for more information.  */
```

Comment:
This license block has a weird format.
Could you re-write it so that it looks similar to the other files?

Reply:
I did a ctrl+c ctrl+v on the license from another file? What's wrong? Indentation?

Reply:
At a glance it seems different.

I see that there are empty lines splitting the paragraphs in the other files.  Maybe that's the only difference.

PS: I just noticed that the year is 2016.  Please change it to 2017.